examples/default/ios/Podfile.lock

```diff
@@ -793,6 +843,6 @@ SPEC CHECKSUMS:
   Yoga: 8796b55dba14d7004f980b54bcc9833ee45b28ce
   YogaKit: f782866e155069a2cca2517aafea43200b01fd5a
 
-PODFILE CHECKSUM: d169b4508f413ff5d69cdf38428960e0828e6282
+PODFILE CHECKSUM: ad82a989387dcdb1ce80667f16b998a2644cf0b2
 
-COCOAPODS: 1.12.0
+COCOAPODS: 1.14.3
```

Comment:
Let's revert this as we did not upgrade to this version yet.

Reply:
fixed